libguard/guard_interface.cpp

```diff
@@ -124,6 +124,28 @@ static GuardRecord getHostEndiannessRecord(const GuardRecord& record)
     return convertedRecord;
 }
 
+/**
+ * @brief Function to get the size remaining in GUARD file
+ *
+ * @param[in] size - GUARD file size
+ * @param[in] count - No of records present in GUARD file
+ *
+ * @return NULL on success.
+ *         Throw exception, if GUARD file doesn't have space to add
+ *         new guard record.
+ */
+void fileSizeCheck(size_t size, uint32_t count)
```

Comment:
It is just for internal purpose. Very function we introduce in interface doesnt get exported to external user. This function is an internal check before creating guard record.

Reply:
>It is just for internal purpose.

Then Please move into .cpp and mark it as `static` function 

> Very function we introduce in interface doesnt get exported to external user.
I would say modify those as well if you want. 

In general, the library functions are intended for the user than those functions need to be static in the CPP unit file as static function and if more than one CPP unit needs then make as utils and we should no install that utils header file so that the user won't aware or he knows it is a utility function for library.

Reply:
Its already added in guard_interface.cpp. Changed to static function.